flink-runtime-web/web-dashboard/src/app/pages/job-manager/log-list/job-manager-log-list.component.less

```diff
@@ -19,7 +19,26 @@
 @import "theme";
 
 :host {
-  ::ng-deep .ant-card-body {
-    padding: 0;
+  display: flex;
+  flex: 1;
+  padding: @padding-md;
+
+  ::ng-deep {
+    ::-webkit-scrollbar {
+      display: none;
+    }
+
+    nz-table,
+    nz-spin,
+    cdk-virtual-scroll-viewport,
+    nz-table-inner-scroll,
+    .ant-spin-container,
+    .ant-table {
+      height: 100%;
+    }
```

Comment:
(This all seems very hacky and fragile. It'd be great of ng-zorro exposed an API to achieve this…)